backend/src/test/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceControllerTest.java

```diff
@@ -0,0 +1,160 @@
+    // GIVEN
+    initPatientAndGetLink();
+
+    String dob = _person.getBirthDate().format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+    String newFirstName = "Blob";
+    String newLastName = "McBlobster";
+
+    String requestBody = "{\"patientLinkId\":\"" + _patientLink.getInternalId() + "\",\"dateOfBirth\":\"" + dob
+        + "\",\"data\":{\"firstName\":\"" + newFirstName + "\",\"middleName\":null,\"lastName\":\"" + newLastName
+        + "\",\"birthDate\":\"0101-01-01\",\"telephone\":\"123-123-1234\",\"role\":\"UNKNOWN\",\"email\":null,\"race\":\"refused\",\"ethnicity\":\"not_hispanic\",\"gender\":\"female\",\"residentCongregateSetting\":false,\"employedInHealthcare\":true,\"address\":{\"street\":[\"12 Someplace\",\"CA\"],\"city\":null,\"state\":\"CA\",\"county\":null,\"zipCode\":\"67890\"}}}";
+
+    // WHEN
+    MockHttpServletRequestBuilder builder = put("/pxp/patient").contentType(MediaType.APPLICATION_JSON_VALUE)
+        .accept(MediaType.APPLICATION_JSON).characterEncoding("UTF-8").content(requestBody);
+
+    // THEN
+    mockMvc.perform(builder).andExpect(status().isOk()).andExpect(jsonPath("$.firstName", is(newFirstName)))
+        .andExpect(jsonPath("$.lastName", is(newLastName)));
+  }
+
+//   @Test
+//   public void aoeSubmitCallsUpdate() throws Exception {
+//     // GIVEN
+//     initPatientAndGetLink();
+
+//     String dob = _person.getBirthDate().format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+//     String symptomOnsetDate = "2021-02-01";
+//     String requestBody = "{\"patientLinkId\":\"" + _patientLink.getInternalId() + "\",\"dateOfBirth\":\"" + dob
+//         + "\",\"data\":{\"noSymptoms\":false,\"symptoms\":\"{\\\"25064002\\\":false,\\\"36955009\\\":false,\\\"43724002\\\":false,\\\"44169009\\\":false,\\\"49727002\\\":false,\\\"62315008\\\":false,\\\"64531003\\\":true,\\\"68235000\\\":false,\\\"68962001\\\":false,\\\"84229001\\\":true,\\\"103001002\\\":false,\\\"162397003\\\":false,\\\"230145002\\\":false,\\\"267036007\\\":false,\\\"422400008\\\":false,\\\"422587007\\\":false,\\\"426000000\\\":false}\",\"symptomOnset\":\"" + symptomOnsetDate+"\",\"firstTest\":true,\"priorTestDate\":null,\"priorTestType\":null,\"priorTestResult\":null,\"pregnancy\":\"261665006\"}}";
+
+//     // WHEN
+//     MockHttpServletRequestBuilder builder = put("/pxp/questions").contentType(MediaType.APPLICATION_JSON_VALUE)
+//         .accept(MediaType.APPLICATION_JSON).characterEncoding("UTF-8").content(requestBody);
+
+//     // THEN
+//     mockMvc.perform(builder).andExpect(status().isOk());
+
+//     TestOrder order = _testOrderService.getTestOrder(_patientLink.getTestOrder().getInternalId().toString());
+//     PatientAnswers answers = order.getAskOnEntrySurvey();
+
+//     AskOnEntrySurvey survey = answers.getSurvey();
+//     assertEquals(survey.getNoSymptoms(), false);
+//     assertEquals(survey.getSymptomOnsetDate(), LocalDate.parse("2021-02-01"));
+//   }
```

Comment:
Could this be either uncommented or removed? I don't think other tests in this class are exercising the interaction with the test order service, so I would lean towards the former.

Reply:
Uncommented, yes! I was having some security troubles w/ the tests, Ben is helping me